pkg/config/config.go

```diff
@@ -704,6 +704,7 @@ func InitConfig(config Config) {
 	config.SetKnown("apm_config.obfuscation.remove_stack_traces")
 	config.SetKnown("apm_config.obfuscation.redis.enabled")
 	config.SetKnown("apm_config.obfuscation.memcached.enabled")
+	config.SetKnown("apm_config.obfuscation.sql.new_sql_normalization")
```

Comment:
This seems like it should be a feature flag not a yaml setting. Please search the codebase for DD_APM_FEATURE to get an idea how to use that.

Reply:
In a previous commit, I did use the environment variables features API: https://github.com/DataDog/datadog-agent/pull/5664/commits/d6448e4fc067a5d4ac0982c3052671012e6d85d9#diff-442b5815544307bac8cdd23db1de09c5R192

But I moved it to the agent config because we have been getting a lot of pushback on the ease of onboarding customers. Having some of these settings (such as `deep_database_monitoring`) in agent config and others in environment variables is a confusing onboarding experience and I think I'll get the same pushback when I share documentation.

Ideally this will be a short-lived option that exists only through private alpha before it is removed and defaulted to true.

Reply:
I understand but this is exactly what feature flags are for. Furthermore, customers already configure the agent by means of env. vars, so this would just be more of that. Alternatively, we can add another config option called “features” which takes a list of features. 

If you like, it’s also ok to bring in more people and get their feedback. 

Ultimately, something like this has no place in config in my opinion but happy to discuss.

Reply:
Ok I'll revert back to using config.HasFeature. We're drafting the onboarding documentation tomorrow so we'll solicit feedback on it and go from there.

Reply:
Thanks Justin! Let’s work in parallel with that. Happy to chip in there too if you think it helps.